### PR TITLE
Add hosts for usa migration

### PR DESCRIPTION
**Pull request by andrewpbrett**

### CODE REVIEW

File: inventory/hosts [hunk 1]
```diff
@@ -108,11 +108,12 @@ uk-staging2
 [us-prod]
 openfoodnetwork.net ansible_host=52.3.171.208
 
-[us-staging]
+[us-migrate]
+migrate.openfoodnetwork.net ansible_host=64.227.102.29
```
Comment (sauloperez) [inline inventory/hosts:112]: so we'll keep both domains during the migration and then do switch over?

File: inventory/hosts [hunk 2]
```diff
@@ -108,11 +108,12 @@ uk-staging2
...
 
 [us:children]
 us-prod
-us-staging
```
Comment (sauloperez) [inline inventory/hosts:115]: it doesn't exist anymore? just curious.

File: inventory/hosts (hunk 1 above)
Comment (andrewpbrett) [reply inventory/hosts:112]: Yeah, I was basing it on this: https://github.com/openfoodfoundation/ofn-install/pull/611/files

File: inventory/hosts (hunk 2 above)
Comment (andrewpbrett) [reply inventory/hosts:115]: I don't think it ever existed? I made one very briefly but it wasn't needed.

**Review by sauloperez — APPROVED** [review]

**Pull request merged**